src/WingetCreateCore/Common/GitHub.cs

```diff
@@ -59,6 +59,18 @@ public static async Task<string> GetGitHubAppInstallationAccessToken(string gitH
             var installation = await github.GitHubApps.GetRepositoryInstallationForCurrent(wingetRepoOwner, wingetRepo);
             var response = await github.GitHubApps.CreateInstallationToken(installation.Id);
             return response.Token;
+        }
+
+        /// <summary>
+        /// Gets the latest release tag name of winget-create.
+        /// </summary>
+        /// <returns>Latest release tag name.</returns>
+        public static async Task<string> GetLatestRelease()
+        {
+            var github = new GitHubClient(new ProductHeaderValue(UserAgentName));
+            var releases = await github.Repository.Release.GetAll("microsoft", "winget-create");

```

Comment:
Why not use GetLatest? Should be faster too

Reply:
Changed to using GetLatest